src/content/app/entity-viewer/state/api/entityViewerThoasSlice.ts

```diff
@@ -49,10 +51,11 @@ import {
 type GeneQueryParams = { genomeId: string; geneId: string };
 type ProductQueryParams = { productId: string; genomeId: string };
 
-const entityViewerThoasSlice = thoasApiSlice.injectEndpoints({
+const entityViewerThoasSlice = graphqlApiSlice.injectEndpoints({
   endpoints: (builder) => ({
     genePageMeta: builder.query<GenePageMeta, GeneQueryParams>({
       query: (params) => ({
+        url: config.thoasBaseUrl,
         body: genePageMetaQuery,
         variables: params
```

Comment:
How about creating a function that returns this object? It seems quite repetitive in this file and has the same `url` value.

Reply:
It does look repetitive; but I do not know how to make it less repetitive (and not to break redux-toolkit-query in the process). This is my least favourite part of redux-toolkit-query. You are welcome to try and improve it 😃